config/configgrpc/configgrpc.go

```diff
@@ -39,6 +39,7 @@ import (
 	"go.opentelemetry.io/collector/client"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config/configauth"
+	"go.opentelemetry.io/collector/config/configcompression"
```

Comment:
No strong feelings, but I think I would prefer confignet, if we believe we might need another shared configs than compression.

Reply:
I don't mind using the `confignet`, @tigrannajaryan @open-telemetry/collector-approvers please give at least one opinion for `configcompression` vs `confignet`

Reply:
I find `configcompression` more meaningful as to what the contents of the package are today. `confignet` doesn't make me think of compression necessarily, and it would be possible for these options to be used outside of network transport (ie for compressing data before saving it to a datastore) in which case it would be confusing for the option to be in `confignet`. It's not a strong preference but just what comes to mind off top my head.